front/app/components/ConditionalWrapper.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactElement, ReactNode } from 'react';
```

Comment:
I'm a little confused by this component - what is the value of introducing it? Also - ideally everything added to our main component folder should have a unit test.